main.py

```diff
@@ -252,6 +252,32 @@ def list(self, dpid=None):
 
         return jsonify(switch_flows)
 
+    @listen_to('kytos.flow_manager.flows.(install|delete)')
+    def event_add_flow(self, event):
```

Comment:
Please use a name that better reflects what this method can do.

```suggestion
    def event_flows_install_delete(self, event):
```